src/cljs/main/broadfcui/common/entity_table.cljs

```diff
@@ -113,22 +116,25 @@
                                          :else ((:attribute-renderer props) attr-value)))})
                           attributes))
                :column-defaults (get (:column-defaults props) (some-> selected-entity-type name))
-               :on-row-click (:on-row-click props)}
+               :on-row-click (:on-row-click props)
+               :on-column-change (:on-column-change props)}
               :toolbar
-              {:items (cons [comps/FilterGroupBar
-                             {:filter-groups (map (fn [entity-type]
-                                                    {:text (name entity-type)
-                                                     :count-override (get-in entity-metadata [entity-type :count])})
-                                                  entity-types)
-                              :selected-index (:selected-filter-index @state)
-                              :on-change (fn [index _]
-                                           (let [selected-entity-type (nth entity-types index)]
-                                             (swap! state assoc
-                                                    :selected-filter-index index
-                                                    :selected-entity-type selected-entity-type)
-                                             (when-let [f (:on-entity-type-selected props)]
-                                               (f selected-entity-type))))}]
-                            (:toolbar-items props))
+              {:items
```

Comment:
This should probably be a follow-on PR and it's my fault, but `items` is no longer an appropriate name for this key. It's a function, so it should start with a verb. I suggest `get-items`.